pepper-apis/src/main/java/org/broadinstitute/ddp/route/GetActivityInstanceStatusTypeListRoute.java

```diff
@@ -1,25 +1,21 @@
 package org.broadinstitute.ddp.route;
 
 import org.broadinstitute.ddp.db.TransactionWrapper;
-import org.broadinstitute.ddp.db.dao.JdbiActivityInstanceStatusType;
+import org.broadinstitute.ddp.db.dao.JbdiActivityInstanceStatusTypeCached;
 import org.broadinstitute.ddp.security.DDPAuth;
 import org.broadinstitute.ddp.util.RouteUtil;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 import spark.Request;
 import spark.Response;
 import spark.Route;
 
 public class GetActivityInstanceStatusTypeListRoute implements Route {
 
-    private static final Logger LOG = LoggerFactory.getLogger(GetActivityInstanceStatusTypeListRoute.class);
-
     @Override
     public Object handle(Request request, Response response) {
         return TransactionWrapper.withTxn(
                 handle -> {
                     DDPAuth ddpAuth = RouteUtil.getDDPAuth(request);
-                    return handle.attach(JdbiActivityInstanceStatusType.class)
+                    return handle.attach(JbdiActivityInstanceStatusTypeCached.class)
```

Comment:
Interesting. Shouldn't this be `new JdbiActivityInstanceStatusTypeCached(handle)` instead?

Reply:
Not interesting. Just incorrect code. 993efd77e4d38758c42d871ecfe7d1a49ed22d20